src/ms-review.tex

```diff
@@ -510,9 +510,7 @@ \section{Gamma-ray Data Analysis}
   migration, $\mu=\frac{E}{E_{\rm true}}$.
 \end{itemize}
 
-%% CN: except for the figures we never introduced what the DL3 or the Data Level acronym is
-%% maybe add (see Fig.~1)
-\gammaray data at the Data Level 3 (DL3) therefore consist of lists of \gammaray-like events and their
+DL3 \gammaray data therefore consist of lists of \gammaray-like events and their
```

Comment:
```suggestion
The DL3 represent the high-level data from which the analysis previously described starts. This data level therefore contains lists of \gammaray-like events and their
```

Reply:
Then it would transition smoothly in the following subsections, where we say that the Gammapy workflow starts from DL3 data.

Reply:
The whole section is restructured in https://github.com/adonath/gammapy-v1.0-paper/pull/1. I think this basically includes your comment. But I'll ask one last time for feedback once I have merged the PRs.